tests/x11/tracker/tracker_by_command.pm

```diff
@@ -32,9 +32,16 @@ sub run {
         script_run "tracker-search newfile";
     }
     else {
-        my $trackercmd = (is_sle('<16') or is_leap('<16.0')) ? 'tracker' : 'tracker3';
+        my $trackercmd = (is_sle('<=15-sp3') or is_leap('<=15.3')) ? 'tracker' : 'tracker3';
         script_run "$trackercmd search emptyfile";
-        assert_screen('tracker-cmdsearch-emptyfile');
+        assert_screen([qw(tracker-cmdsearch-emptyfile tracker-cmdsearch-noemptyfile)]);
+        if (match_has_tag 'tracker-cmdsearch-noemptyfile') {
```

Comment:
Is this 100% reproducible? It might be a race condition.

Reply:
The "tracker-cmdsearch-noemptyfile" can't be reproduced 100% if you restart the test suite multiple times.

From https://openqa.opensuse.org/tests/1913344#next_previous and https://openqa.opensuse.org/tests/1913431#next_previous, 
we can see it happens on TW aarch64 Wayland more often than x11.

Reply:
I might be able to explain this. In the wayland test, the `flatpak` module fails, so it reverts back to the VM state at `consoletest_setup`. At that point, tracker is probably not up and running.

If this theory is correct, the failure should disappear once the `flatpak` failure is fixed.

Reply:
Sounds reasonable and here is another evidence for your theory https://openqa.opensuse.org/tests/1912851.
And that also explains why the tracker test case didn't get "tracker-cmdsearch-noemptyfile" after I added `EXCLUDE_MODULES=flatpal` when cloning the job.

Since there's no update for the flatpak bug for more than 5 months, I'd like to say we keep using this softfailure

Reply:
> Since there's no update for the flatpak bug for more than 5 months, I'd like to say we keep using this softfailure

I'd rather see the root cause worked around, which is either making sure that tracker is started, or waiting until tracker indexed that file. `wait_still_screen` does neither, it's just waiting some random time.